src/components/common/Modal/index.jsx

```diff
@@ -0,0 +1,24 @@
+import ReactDom from 'react-dom';
+import PropTypes from 'prop-types';
+import s from './Modal.module.scss';
+
+export const Modal = ({ open, closeModal }) => {
+  if (!open) return null;
+  return ReactDom.createPortal(
+    <>
+      <div className={s.modalOverlay} onClick={closeModal}>
+        <div className={s.modalLeftContent}>
+          <p>Modal</p>
+        </div>
+      </div>
+    </>,
+    document.getElementById('root')
```

Comment:
if we change the id of the root element, this will stop working. change it to body